README.rst

```diff
@@ -0,0 +1,63 @@
+|Tests Status| |Coverage|
+
+`aehmc` provides MCMC sampling algorithms written in `Aesara <https://github.com/pymc-devs/aesara>`_.
+
+Features
+========
+- Sample from an (unnormalized) probability distribution using Hamiltonian Monte
+  Carlo and the No U-Turn Sampler.
+
+Example
+=======
+
+.. code-block:: python
+
+  import aesara
+  from aesara import tensor as at
+  from aesara.tensor.random.utils import RandomStream
+
+  from aeppl import joint_logprob
+
+  from aehmc import hmc as hmc
+
+  # A simple normal distribution
+  Y_rv = at.random.normal(0, 1)
+
+  def logprob_fn(y):
+      logprob = joint_logprob(Y_rv, {Y_rv: y})
+      return logprob
+
+  # Build the transition kernel
+  srng = RandomStream(seed=0)
+  kernel = hmc.kernel(
+    srng,
+    logprob_fn,
+    step_size=1e-3,
+    inverse_mass_matrix=1.0,
+    num_integration_steps=10,
+  )
+  
+  # Compile a function that updates the chain
+  Y_at = Y_rv.clone()
+  potential_energy = -logprob_fn(Y_at)
+  potential_energy_grad = aesara.grad(potential_energy, wrt=Y_at)
+
+  next_step = kernel(y, potential_energy, potential_energy_grad)
+  print(next_step.eval({Y_at: 0}))
+
+
+Installation
+============
+
```

Comment:
```suggestion
The latest release of AeHMC can be installed from PyPI using ``pip``:

::
    pip install aehmc

Or via conda-forge:

::
    conda install -c conda-forge aehmc
    
```

Reply:
AeHMC cannot be currently installed via pip, which is why I removed this from the README.

Reply:
Ah. then I'll set that up.

Reply:
OK, `pip install` should work now.  Also, the Conda setup is in progress (https://github.com/conda-forge/staged-recipes/pull/15767), so you can keep that in.

Reply:
Don't forget to add these `pip install` instructions.

Reply:
Done.